server_admin/topics/identity-broker/first-login-flow.adoc

```diff
@@ -91,4 +91,23 @@ In this case, automatic user creation should be turned off. To disable user crea
 * set `Confirm Link Existing Account` to `DISABLED`.
 
 This configuration also implies that Keycloak itself won't be able to determine which internal account would correspond to the external identity.
-Therefore, the `Verify Existing Account By Re-authentication` authenticator will ask the user to provide both username and password.
\ No newline at end of file
+Therefore, the `Verify Existing Account By Re-authentication` authenticator will ask the user to provide both username and password.
+
+==== Detect Existing User First Login Flow
+In order to configure a first login flow in which :
+
+  - only users already registered in this realm can login,
+  - users are automatically linked without being prompted,
+
+create a new flow with the following two authenticators:
+
+Detect Existing Broker User::
+This authenticator ensures that unique users are handled. Set the authenticator requirement to `Mandatory`.
+
+Automatically Set Existing User::
+Automatically sets an existing user to the authentication context without any verification. Set the authenticator requirement to `Mandatory`.
+
+You have to set the `First Login Flow` of the identity provider configuration to that flow.
+You could set the also set `Sync Mode` to `force` if you want to update the user profile (Last Name, First Name...) with identity provider attributes.
+
+NOTE: This flow can be used if you want to delegate identity to other identity providers (i.e. github, facebook ...) but you want to manage which users that can login.
```

Comment:
add "the" before "identity"
Always use "such as" instead of "i.e."  (We avoid Latin)
login should be "log in" since it is a verb.  Login is a noun.